pkg/reconcile/pipeline/handler/collect/impl_test.go

```diff
@@ -388,6 +388,62 @@ var _ = Describe("Collect Binding Data", func() {
 			shouldRetry(pipeline.HandlerFunc(collect.BindingItems), collect.ErrorReadingBindingReason, err)
 		})
 
+		Context("on invalid sourceValue in sliceOfMap elements", func() {
+			var (
+				mockCtrl *gomock.Controller
+			)
+
+			BeforeEach(func() {
+				mockCtrl = gomock.NewController(GinkgoT())
+			})
+
+			AfterEach(func() {
+				mockCtrl.Finish()
+			})
```

Comment:
this is not needed because they are already declared in the outer context. With that, we can even remove introduced `Context`, because it does not contain any additional customizations.